const num1 = 20;
const num3 = 10;

console.log(num1);

try{
    console.log(num2);
} catch{
    console.log('error');
}
console.log(num3);
